### PR TITLE
feat(dashboards): add linked entities to nr1 dashboard widgets

### PR DESCRIPTION
**Pull request by sanderblue**
Related to: [newrelic/terraform-provider-newrelic #1128](https://github.com/newrelic/terraform-provider-newrelic/issues/1128)

### CODE REVIEW

File: pkg/dashboards/dashboard.go [hunk 1]
```diff
@@ -97,7 +97,10 @@ const getDashboardEntityQuery = `query ($guid: EntityGuid!) {
             layout { column height row width }
             title
             visualization { id }
-            id
+						linkedEntities {
+							__typename
+							guid
```
Comment (sanderblue) [inline pkg/dashboards/dashboard.go:102]: Should we include more fields here?

**Review by ctrombley — DISMISSED** [review]

File: pkg/dashboards/dashboard.go (hunk 1 above)
Comment (jthurman42) [reply pkg/dashboards/dashboard.go:102]: Need to keep `id`, which appears to have been dropped.  Can you convert to spaces for consistency within the string?

`GUID` is the only required input field for adding the link to a widget, `name` and `accountId` would be useful for validation / human consumption.  The data set would be rather small on fetch, so adding more fields should have minimal impact if any.  Maybe something like this:

```
            linkedEntities {
              __typename
              guid
              name
              accountId
              tags {
                key
                values
              }
              ... on DashboardEntityOutline {
                dashboardParentGuid
              }
            }
```

Dashboard entities are the only supported type today, might be helpful to have the parent GUID, might never be used.

**Review by jthurman42 — DISMISSED** [review]
Excellent! Ship or enhance.

File: pkg/dashboards/dashboard.go (hunk 1 above)
Comment (sanderblue) [reply pkg/dashboards/dashboard.go:102]: Oh crap, I didn't even realize I removed that. Good catch.

Comment (sanderblue) [reply pkg/dashboards/dashboard.go]: Thanks for the suggestions. Updated

**Review by ctrombley — APPROVED** [review]
:shipit:

**Pull request merged**